docs/index.asciidoc

```diff
@@ -546,7 +547,9 @@ default id is used instead.
 query::
 A SQL SELECT statement that is executed to achieve the lookup. To use
 parameters, use named parameter syntax, for example
-`"SELECT * FROM MYTABLE WHERE ID = :id"`.
+`"SELECT * FROM MYTABLE WHERE ID = :id"`. Alternatively could be also use
```

Comment:
"Alternatively the `?` sign could be used as prepared statement parameters, in which case the `prepared_parameters` is used to populate the values." 
(maybe "... provide the values") sounds better - not sure.

Reply:
What about, "Alternatively, the `?` sign can be used as a prepared statement parameter, in which case the `prepared_parameters` array is used to populate the values